net-mail/getmail/getmail-6.03.ebuild

```
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_REQ_USE="ssl?"
+
+DISTUTILS_USE_SETUPTOOLS=no
+inherit distutils-r1
+
+DESCRIPTION="A mail retriever with reliable Maildir and mbox delivery"
+HOMEPAGE="http://www.getmail6.org/"
+SRC_URI="https://github.com/getmail6/getmail6/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~ppc-macos ~x86-macos ~x64-solaris"
```

Let's drop the prefix keywords for now, 
```suggestion
KEYWORDS="~amd64 ~ppc ~x86"
```